frontend_tests/node_tests/buddy_data.js

```diff
@@ -2,10 +2,15 @@ const _page_params = {};
 
 set_global('page_params', _page_params);
 set_global('i18n', global.stub_i18n);
+set_global('$', global.make_zjquery());
 zrequire('people');
 zrequire('presence');
 zrequire('util');
 zrequire('user_status');
+
+// var noop = function () {};
+// $.fn.tooltip = noop;
```

Comment:
Clean this up?

Reply:
Removed the comments and pushed the changes.